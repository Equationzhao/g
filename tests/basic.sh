output="$(g -term-width 200 --no-config --no-update --icons tests/test_data)"
echo "$output" | diff - tests/basic.stdout
