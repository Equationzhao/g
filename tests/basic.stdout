[0;94m 14.dir[0m   [38;5;153m 1.txt[0m    [0;37m 10.json[0m  [0;37m 11.xml[0m   [0;37m 12.yml[0m   [0;93m 13.md[0m    [38;5;158m 2.c[0m      [38;5;158m 3.cpp[0m    [38;5;158m 4.py[0m     [38;5;158m 5.java[0m   [0;92m 6.sh[0m     [38;5;158m 7.html[0m   [38;5;158m 8.css[0m    [38;5;158m 9.js[0m     
